### PR TITLE
Fixes #2367. Resume pause in Never Ending Reddit when clicking "Open Next Page"

### LINKED ISSUE #2367: An option to automaticall resume loading in Never Ending Reddit
Can such a feature be added? 
I mean, currently my "workflow" with RES is like this: 
1. Scroll down 7 pages (my value of `pauseAfterEvery` option in `Never Ending Reddit`);
2. Press "Open Next Page";
3. In the new page press "Resume", so the Never Ending Reddit works again;
4. Repeat.

If there was an option to automatically resume scrolling when a new page is opened, I wouldn't have to press "Resume" manually every time. What do you guys think?

### PR DESCRIPTION
**Pull request by helarqjsc**
After this PR, clicking "Open Next Page" button manually will always resume pause. Is that alright? I can add an option if you think it should be possible to disable such behavior, but to me it seems that there is no reason to have Never Ending Reddit be paused on a new page.

### CODE REVIEW

**Comment by helarqjsc** [comment]
Any reason why it hasn't been merged yet? Is there a problem with the code? It's been half a year, but #2367 issue still persists, even though there is a pull requests that fixes it.

**Comment by andytuba** [comment]
Sorry about the terrible delay. This was targeted to 4.6.2 milestone when I started putting a freeze on 4.6.0. After 4.6.0 (upcoming) and 4.6.1 (hotfix), I plan to merge 4.6.2 PRs like this into the 4.6 branch and subsequently into master.

If someone from RES core team can review and test this before 4.6.0 release, I'll consider merging it earlier.

tl;dr: backed up release schedule

**Comment by erikdesjardins** [comment]
Merged in #2897.

Thanks @helarqjsc! (sorry about the delay)
